services/workflows-service/src/env.ts

```diff
@@ -90,6 +90,12 @@ export const serverEnvSchema = {
   IN_MEMORIES_SECRET_ACQUIRER_ID: z.string().optional(),
   IN_MEMORIES_SECRET_PRIVATE_KEY: z.string().optional(),
   IN_MEMORIES_SECRET_CONSUMER_KEY: z.string().optional(),
+  REDIS_HOST: z.string(),
+  REDIS_PASSWORD: z.string(),
+  REDIS_PORT: z
+    .string()
+    .default('6379')
```

Comment:
Does it make sense to set it as the default?
```suggestion
    .default('7381')
```